R/seroprevalence_data.R

```diff
@@ -139,6 +139,32 @@ prepare_bin_data <- function(serodata) {
   return(xx)
 }
 
+#' Function that returns the probability of being seropositive for age-varying
+#' FoI including seroreversion
```

Comment:
force-of-infection (I think good to not use acronyms so people don't get confused)

Reply:
Also I think no need to start function docs with "Function that..." since almost everything we document is a function. Instead, I would reword this as "Calculate probability that an individual of known age is seropositive given an age-varying force-of-infection"

Reply:
I'll address this in a separate issue (#163) since it affects other functions other than the data simulation functions.

Reply:
Makes sense! Thanks